update-website

```diff
@@ -1,2 +1,3 @@
+export LC_ALL=POSIX
```

Comment:
Because (as far as I see) it is the way to set the locale for the rest of the script (which otherwise is run in the locale of the shell, which in my case isn't English).
I am adding a comment.

Reply:
Let me know if the comment is too long or not clear enough.

Reply:
You are right. I was missing this point. Thanks!